base_wamas/tests/test_base_wamas.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2023 Jacques-Etienne Baudoux (BCIM) <je@bcim.be>
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import filecmp
+import os
+import tempfile
+
+from odoo.tests.common import TransactionCase
+from odoo.tools import file_open, file_path
+
+
+class TestBaseWamas(TransactionCase):
```

Comment:
It would be nice to have full test coverage, possibly via unit tests

Reply:
sure, I'll do it, if I finish the other main functions